Extension/src/LanguageServer/extension.ts

```diff
@@ -1120,10 +1120,16 @@ function reportMacCrashes(): void {
     }
 }
 
+let previousMacCrashData: string;
+let previousMacCrashCount: number = 0;
 function logCrashTelemetry(data: string): void {
```

Comment:
is `logCrashTelemetry` only for Mac or will it also get used on other platforms?

Reply:
Yes, it's just for Mac -- logCrashTelemetry method should probably be renamed logMacCrashTelemetry.

Reply:
I renamed the methods -- the code should probably be moved to it's own file, but that could be left for later.